contrib/packs/actions/install.meta.yaml

```diff
@@ -17,6 +18,10 @@
       type: "string"
       default: "actions,aliases,sensors,triggers"
       description: "Possible options are all, triggers, sensors, actions, rules, aliases."
+    upgrade:
+      type: "boolean"
+      default: false
+      description: "Upgrade the pack if already installed."
```

Comment:
Installing + Downloading over and over again (even if it reaches the same state) is not considered really "Idempotent" in configuration management.
If resources is in desired state - we shouldn't repeat the same operation again.